create-container.sh

```diff
@@ -98,27 +112,35 @@ read -r ssh_key < "$ssh_path"
 echo "Copying system user's SSH public key to 'root' user in container"
 sudo lxc-attach -n "$NAME" -- /bin/bash -c "/bin/mkdir -p /root/.ssh && echo $ssh_key > /root/.ssh/authorized_keys"
 
-# Find `uid` of project directory
-project_user=$(stat -c '%U' "$PROJECT_PATH")
-project_uid=$(id -u "$project_user")
+# User management related with projects folder
+# If PROJECT_PATH is not set, use the defaults project_uid and project_gid
+if  [ -v PROJECT_PATH ] ; then
+  # Find `uid` of project directory
+  project_user=$(stat -c '%U' "$PROJECT_PATH")
+  project_uid=$(id -u "$project_user")
 
-# Find `gid` of project directory
-project_group=$(stat -c '%G' "$PROJECT_PATH")
-project_gid=$(id -g "$project_group")
+  # Find `gid` of project directory
+  project_group=$(stat -c '%G' "$PROJECT_PATH")
+  project_gid=$(id -g "$project_group")
+fi
 
-# Delete existing user with same uid and gid of project directory
-existing_user=$(sudo lxc-attach -n "$NAME" -- id -nu "$project_uid" 2>&1)
-sudo lxc-attach -n "$NAME" -- /usr/sbin/userdel -r "$existing_user"
+# User management
+# If DEVENV_USER or DEVENV_GROUP is not set, the defaults user is not deleted.
```

Comment:
I would say that if `DEVENV_USER` or `DEVENV_GROUP` are set we delete the default user. Double negations are always difficult to understand.

Also, I'm not sure about the `or`. Shouldn't it be an `and`?

Reply:
Yes, I changed the condition to be more explicit and it checks also if `project_uid` and project_gid` are set.

I remove the default values of `project_uid` and roject_gid` as you say in #12